project_stock/models/project_task.py

```diff
@@ -0,0 +1,236 @@
+        if not location:
+            warehouse = self.env["stock.warehouse"].search(
+                [("company_id", "=", self.env.company.id)], limit=1
+            )
+            location = warehouse.lot_stock_id
+        return location.id if location else False
+
+    @api.model
+    def _get_default_location_dest_id(self):
+        location = False
+        if self.env.context.get("default_project_id"):
+            project = self.env["project.project"].browse(
+                self.env.context.get("default_project_id")
+            )
+            location = project.location_dest_id
+        if not location:
+            warehouse = self.env["stock.warehouse"].search(
+                [("company_id", "=", self.env.company.id)], limit=1
+            )
+            location = warehouse.lot_stock_id
+        return location.id if location else False
+
+    procurement_group_id = fields.Many2one(
+        comodel_name="procurement.group", string="Procurement Group", copy=False
+    )
+    scrap_ids = fields.One2many(
+        comodel_name="stock.scrap", inverse_name="task_id", string="Scraps"
+    )
+    scrap_count = fields.Integer(
+        compute="_compute_scrap_move_count", string="Scrap Move"
+    )
+    move_raw_ids = fields.One2many(
+        comodel_name="stock.move",
+        inverse_name="raw_material_task_id",
+        string="Stock Moves",
+        copy=False,
+        domain=[("scrapped", "=", False)],
+    )
+    use_stock_moves = fields.Boolean(related="stage_id.use_stock_moves")
+    lock_stock_moves = fields.Boolean(related="stage_id.lock_stock_moves")
+    stock_state = fields.Selection(
+        selection=[
+            ("pending", "Pending"),
+            ("confirmed", "Confirmed"),
+            ("assigned", "Assigned"),
+            ("done", "Done"),
+            ("cancel", "Cancel"),
+        ],
+        compute="_compute_stock_state",
+    )
+    picking_type_id = fields.Many2one(
+        comodel_name="stock.picking.type",
+        string="Operation Type",
+        domain="[('code', '=', 'ps_operation'), ('company_id', '=', company_id)]",
+        default=_get_default_picking_type,
+        index=True,
+        check_company=True,
+    )
+    location_src_id = fields.Many2one(
+        comodel_name="stock.location",
+        string="Source Location",
+        domain="[('usage','=','internal')]",
+        default=_get_default_location_src_id,
+        index=True,
+        check_company=True,
+    )
+    location_dest_id = fields.Many2one(
+        comodel_name="stock.location",
+        string="Destination Location",
+        domain="[('usage','=','internal')]",
+        default=_get_default_location_dest_id,
+        index=True,
+        check_company=True,
+    )
+    unreserve_visible = fields.Boolean(
+        string="Allowed to Unreserve Inventory",
+        compute="_compute_unreserve_visible",
+        help="Technical field to check when we can unreserve",
+    )
+    analytic_account_id = fields.Many2one(
```

Comment:
```suggestion
    stock_analytic_account_id = fields.Many2one(
```

And add also `stock_analytic_tag_ids`